apps/pdc-frontend/src/app/[locale]/(rootLayout)/products/[slug]/page.tsx

```diff
@@ -298,23 +299,25 @@ const Product = async ({ params: { locale, slug } }: ProductProps) => {
         }}
         Link={Link}
       />
-      <main id="main">
-        <Article>
-          <Heading level={1}>{product?.attributes?.title}</Heading>
-          {product?.attributes?.content && (
-            <Markdown imageUrl={getImageBaseUrl()} priceData={priceData} locale={locale}>
-              {product?.attributes?.content}
-            </Markdown>
-          )}
-          {product?.attributes?.sections && product.attributes.sections.length > 0 && (
-            <Sections
-              sections={product.attributes.sections as ProductSectionsDynamicZone[]}
-              locale={locale}
-              priceData={priceData}
-            />
-          )}
-        </Article>
-      </main>
+      <RichText>
+        <main id="main">
+          <Article>
+            <Heading level={1}>{product?.attributes?.title}</Heading>
+            {product?.attributes?.content && (
+              <Markdown imageUrl={getImageBaseUrl()} priceData={priceData} locale={locale}>
+                {product?.attributes?.content}
+              </Markdown>
+            )}
+            {product?.attributes?.sections && product.attributes.sections.length > 0 && (
+              <Sections
+                sections={product.attributes.sections as ProductSectionsDynamicZone[]}
+                locale={locale}
+                priceData={priceData}
+              />
+            )}
+          </Article>
+        </main>
+      </RichText>
```

Comment:
Ik zou het omdraaien: `<main><article><RichText>...</RichText></article></main>`. Op die manier kun je ook nog iets anders dan rich text in de main content hebben, nu is alles verplicht rich text met de bijbehorende margins.